projects/packages/publicize/src/class-publicize-base.php

```diff
@@ -1441,5 +1453,6 @@ protected static function build_sprintf( $args ) {
  * @return string
  */
 function publicize_calypso_url() {
+	_deprecated_function( __METHOD__, 'jetpack-10.9', 'Publicize::publicize_connections_url' );
 	return Redirect::get_url( 'calypso-marketing-connections', array( 'site' => ( new Status() )->get_site_suffix() ) );
```

Comment:
Should we make this call `publicize_connections_url`?

Reply:
@danielpost  suggested that as well, but I wonder what downside we will have to creating the global $publicize object twice. It won't have any issues with the new auto-loader package, but the decision to retain the `publicize_calypso_url` was done more based on what we have in the jetpack/modules directory for use with the wpcom code. There, we might have some side effects, if I instantiate a new publicize class, which is why I let the function continue its usual course.

Reply:
Oh I see! Yes, we could maybe use/check for the global instance, but it's probably safer to leave it as is.